lib/manager/helmv3/extract.ts

```diff
@@ -0,0 +1,76 @@
+import is from '@sindresorhus/is';
+import yaml from 'js-yaml';
+import * as datasourceHelm from '../../datasource/helm';
+import { logger } from '../../logger';
+import { SkipReason } from '../../types';
+import { ExtractConfig, PackageDependency, PackageFile } from '../common';
+
+export function extractPackageFile(
+  content: string,
+  fileName: string,
+  config: ExtractConfig
+): PackageFile | null {
+  let chart;
```

Comment:
Please add a type for this

Reply:
Done. But I am not sure if the way I've done it is ideal. Should it be moved outside the scope of the method?